docs/source/dev/dockerfile/dockerfile.rst

```diff
@@ -0,0 +1,34 @@
+Dockerfile
+====================
+
+-  Visualization of the multi-stage Dockerfile
+
+   .. figure:: ../../assets/dev/dockerfile-stages-dependency.png
+      :alt: query
+      :width: 100%
+      :align: center
+
+   Made using: https://github.com/patrickhoefler/dockerfilegraph
+
+   Commands to regenerate it:
+
+   .. code:: bash
+
+      dockerfilegraph -o png --legend --dpi 200 --max-label-length 50
```

Comment:
Should we make it clear that this should be run at top level of this repository where `Dockerfiles` is present at?

Reply:
Done. I also added the `--filename` flag to make it obvious which file we're generating the graph for.